src/compiler/crystal/command.cr

```diff
@@ -480,8 +478,8 @@ class Crystal::Command
     opts.on("-s", "--stats", "Enable statistics output") do
       compiler.progress_tracker.stats = true
     end
-    opts.on("--no-progress", "Disable progress output") do
-      compiler.progress_tracker.progress = false
+    opts.on("-p", "--progress", "Enable progress output") do
```

Comment:
The only other crystal command using `-p` is `crystal play` (for port), which doesn't take compiler options, so this is safe.